projects/docs/docs/plugins/using-plugins.md

```diff
@@ -63,6 +63,10 @@ PluginLocation.git(url: "https://url/to/plugin.git", sha: "e34c5ba")
 
 The [`Plugin.swift`](plugins/creating-plugins.md) manifest in a git repository should be located at the root of the repository.
 
+### Fetching plugins
+
+After plugins have been declared in `Config.swift`, you need to fetch them by running [`tuist fetch`](commands/dependencies.md#fetching). 
```

Comment:
Instead of adding a section, I would just add the sentence at the end of `Example` below

Reply:
I changed 🕺🏼

Reply:
I changed 🕺🏼